logfire/_internal/integrations/aiohttp_client.py

```diff
@@ -81,6 +84,39 @@ def capture_headers(self):
         if self.response:
             capture_request_or_response_headers(self.span, self.response.headers, 'response')
 
+    def capture_body_if_text(self, attr_name: str = 'http.response.body.text') -> None:
+        response = self.response
+        if response is None:
+            return
+
+        original_read = response.read
+
+        @functools.wraps(original_read)
+        async def read() -> bytes:
+            if getattr(response, '__logfire_body_captured', False):
```

Comment:
I think you could just set an attribute on `self` instead of the response

Reply:
Oh yeah, I can remove frozen now that I am not inheriting from TraceRequestEndParams, I'll do that.